patterns/search-big-heading.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Title: Search - Big heading and search form
+ * Slug: twentytwentyfive/search-big-heading
+ * Categories: call-to-action, text
+ * Description: heading with search form
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"metadata":{"name":"Big heading and search form"},"align":"wide","style":{"spacing":{"blockGap":"var:preset|spacing|50","padding":{"top":"140px","bottom":"140px"}}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch"}} -->
```

Comment:
@jenniferfarhat Thanks for the PR! 

Could you please remove meta data?

Reply:
Metadata has been removed.